### PR TITLE
fix(color picker): decorator order

### PR DESCRIPTION
**Pull request by celine-s**

### CODE REVIEW

File: src/components/ColorPicker/ColorInput.tsx [hunk 1]
```diff
@@ -36,30 +36,30 @@ export const ColorInput: FC<ColorInputProps> = ({
...
-                onKeyDown={handleKeyDown}
-                onChange={(event) => onChange && onChange(event.currentTarget.value)}
-                onBlur={onBlur}
-                value={value}
-                type={type}
-                min={min}
-                max={max}
-                size={size}
-            />
+            <div className={`tw-flex ${decorator === "%" ? "tw-flex-row-reverse" : "tw-gap-1 md:tw-pl-0.5"}`}>
```
Comment (Walkeezy) [inline src/components/ColorPicker/ColorInput.tsx]: Nit:
```suggestion
            <div className={merge([
                "tw-flex",
                decorator === "%" ? "tw-flex-row-reverse" : "tw-gap-1 md:tw-pl-0.5",
            ])}>
```

Comment (celine-s) [reply src/components/ColorPicker/ColorInput.tsx]: done 👌🏻:)

File: src/components/ColorPicker/ColorInput.tsx [hunk 2]
```diff
@@ -36,30 +42,35 @@ export const ColorInput: FC<ColorInputProps> = ({
...
-                value={value}
-                type={type}
-                min={min}
-                max={max}
-                size={size}
-            />
+            <div
+                className={merge([
+                    "tw-flex",
+                    decoratorPosition === DecoratorPosition.Right ? "tw-flex-row-reverse" : "tw-gap-1",
```
Comment (NadiaPosch) [inline src/components/ColorPicker/ColorInput.tsx:53]: The thing with the gap is, that the input is currently left aligned. This looks weird if you type in a value < 100, so I'd suggest you right-align the text if the decorator is on the right (see comment below)
```suggestion
                    "tw-flex tw-gap-1",
                    decoratorPosition === DecoratorPosition.Right && "tw-flex-row-reverse",
```

File: src/components/ColorPicker/ColorInput.tsx [hunk 3]
```diff
@@ -36,30 +42,35 @@ export const ColorInput: FC<ColorInputProps> = ({
...
+                ])}
+            >
+                {decorator && (
+                    <div className="tw-flex tw-items-center tw-justify-center tw-text-black-80">{decorator}</div>
+                )}
+                <input
+                    id={useMemoizedId()}
+                    ref={inputElement}
+                    className="tw-grow tw-border-none tw-outline-none tw-bg-transparent tw-hide-input-arrows tw-text-black tw-placeholder-black-60 dark:tw-text-white"
```
Comment (NadiaPosch) [inline src/components/ColorPicker/ColorInput.tsx:62]: `decoratorPosition === DecoratorPosition.Right && "tw-text-right"`

File: src/components/ColorPicker/ColorInput.tsx (hunk 2 above)
Comment (celine-s) [reply src/components/ColorPicker/ColorInput.tsx]: good idea, thanks. changed it

File: src/components/ColorPicker/ColorInput.tsx (hunk 3 above)
Comment (celine-s) [reply src/components/ColorPicker/ColorInput.tsx]: done

**Review by Walkeezy — APPROVED** [review]

**Pull request merged**